### PR TITLE
Improve feature list

### PR DESCRIPTION
**Pull request by vantezzen**
Currently, the feature list in our README is a single, long list - which isn't ideal. 

This PR will:
- sort the features
- add headings to sections of features
- add some features that were previously not part of the list
- add a new "Features" section that more generally describes what features Ferdi has instead of just showing its differences to Franz

A preview of this new list can be found at https://github.com/getferdi/ferdi/tree/feat/better-feature-list#ferdi

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +53,21 @@
...
+- [x] Support for multiple accounts on any service
+- [x] Dark Mode support for all of your services
+- [x] Optional cloud sync to keep your services synchronized between devices
+- [x] Better control over when you get which notifications
+- [x] Cross-platform so you can view your services on all your computers
+- [x] Full proxy support to work in every network environment
+- [x] Ferdi speaks your language: Support for 20+ languages
+
+...and best of all:
+- [x] Its completely free! There are no features hidden behind a paywall
```
Comment (kytwb) [inline README.md]: I would say:

> There are no restrictions on features, no paywalls

File: README.md [hunk 2]
```diff
@@ -92,44 +103,79 @@ Ferdi has three seperate AUR packages you can use:
 
 If you use an AUR Helper e.g. yay, simply install it via `yay -S ferdi`.
 
-## Ferdi-specific Features
+## What makes Ferdi different from Franz?
 
+### Removed unproductive paywalls and other other interruptions
```
Comment (kytwb) [inline README.md]: I would switch from `Removed`/`Added` to `Removes`/`Adds` to keep things consistent and present.

File: README.md [hunk 3]
```diff
@@ -92,44 +103,79 @@ Ferdi has three seperate AUR packages you can use:
...
+
+### Internal changes
+- [x] Upgrading to Electron 8
+- [x] Switch to [`electron-spellchecker`](https://github.com/electron-userland/electron-spellchecker) to improve application size
+- [x] Minifying build files to improve app size
+- [x] Add "npm run prepare-code" command for development to lint and beautify code
+- [x] Fix incorrect body closing tag ([#330](https://github.com/getferdi/ferdi/issues/330))
+- [x] Using CrowdIn to improve i18n
+
+> Thank you to all contributors who helped realizing all these amazing features! You can find a list of our contributors below.
```
Comment (kytwb) [inline README.md]: > Thanks to all our [contributors](#contributors-) who helped realize all these amazing features! 💖

**Comment by vantezzen** [comment]
Thanks. Added your requests and will now merge

**Pull request merged**